Documentation/Building.md

```diff
@@ -1,11 +1,59 @@
 # Building and installing Libevent
 ### Jump to:
+- [Prerequisites](#Prerequisites)
 - [Autoconf](#autoconf)
   - [Flags](#autoconf-flags)
 - [Building on Windows](#building-on-windows)
 - [Building on Unix (With CMake)](#building-on-unix-cmake)
 - [CMake Variables](#cmake-variables)
 
+## Prerequisites
+
+### Linux
+Just install tools using your preferred package manager if you build using autotools:
+```sh
+sudo apt-get install automake autoconf libtool pkg-config
+```
+or build using cmake:
+```sh
+sudo apt-get install cmake
+```
+Doxygen is a tool for generating documentation. Git is used to fetch the package version.  
+Install them if needed:
+```
+sudo apt-get install doxygen git
+```
+If you plan to support for OpenSSL encryption:
```

Comment:
Note about that openssl is required by default would be nice (since there are was some issues regarding this, i.e. people did not have openssl installed or/and properly configured, hence note about proper configuration would be great).

Reply:
I will write a WiKi for it as #703 mentioned.